packages/react-devtools-scheduling-profiler/src/hooks.js

```diff
@@ -53,6 +52,8 @@ export function useBrowserTheme(): void {
         default:
           throw Error(`Unsupported theme value "${theme}"`);
       }
+    } else {
+      updateThemeVariables('light', documentElements);
```

Comment:
@taneliang This should be `theme` not `'light'` right?

Reply:
Ah this was outside of the feature flag conditional. My apologies.

Reply:
Oh I think we want `'light'` to prevent the app from going into dark mode?